packages/datepicker/datepicker.scss

```diff
@@ -103,13 +103,17 @@ $bottom-margin: $component-spacing--large;
     &__year-selector {
         max-width: 30%;
         margin-right: $component-spacing--xxl;
+
+        & > .jkl-datepicker__year-value {
```

Comment:
Do you need this as a nested selector? Why not just put it as a normal BEM-class?